src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const message = document.createElement('div');
+
+message.classList.add('message');
+
```

Comment:
you should create the new blocks for each promise, so better just move this logic to some function